src/main/webapp/i18n/zh-cn/eventActivity.json

```diff
@@ -11,17 +13,17 @@
       "updated": "Event Activity {{ param }} 更新成功",
       "deleted": "Event Activity {{ param }} 删除成功",
       "delete": {
-        "question": "你确定要删除 Event Activity {{ id }} 吗？"
+        "question": "你确定要删除 Event Activity <b>{{ id }}</b> 吗？"
       },
       "detail": {
         "title": "Event Activity"
       },
       "eventId": "Event Id",
-      "startDate": "Start Date",
-      "durationInDay": "Duration In Day",
-      "name": "Name",
-      "description": "Description",
-      "tab": "Timeline"
+      "startDate": "开始日期",
+      "durationInDay": "为期（天数）",
+      "name": "名程",
```

Comment:
typo

Reply:
resolved in dc6059809b4b12e952adc58bc61ae13b8c24de82